tests/build_tests/test_build.py

```diff
@@ -88,30 +88,6 @@ def test_reqs_dependency(self):
         with pytest.raises(OSError):
             save_tasks(task_folder="")
 
-    @pytest.fixture
-    def subdir_reqs_file(self, tmpdir):
-        """
-        It creates a directory structure with a requirements.txt file in it
-
-        :param tmpdir: This is a pytest fixture that creates a temporary directory for us to use
-        :return: The path to the requirements.txt file.
-        """
-        tmpdir.mkdir("tasks").mkdir("subdir").mkdir("subsubdir").join("requirements.txt").write(
-            "magniv"
-        )
-        return str(tmpdir.join("tasks/subdir/subsubdir"))
-
-    def test_subdirreqs_filepath(self, subdir_reqs_file):
```

Comment:
I can't tell, was this moved somewhere or just deleted?

Reply:
Deleted - if you can tell me what functionality it's covering that isn't already covered by the other tests running save_tasks (the assertion here looks pretty meaningless) I'd add it back

Reply:
I think it is supposed to test that nested `requirements.txt` files behave and override as expected but you might be right, not sure this is what happens

Reply:
yeah, I see what you're saying, but I agree I don't think that's what's tested. We're passing in `subdir_reqs_file` not the top-level folder (`tasks`) on L112, We can consider re-adding a test like this, but I think this one is so broken it's not worth keeping